packages/ai/api-report.api.md

```diff
@@ -19,6 +19,12 @@ import type { TLShapePartial } from 'tldraw';
 // @public (undocumented)
 export function asMessage(message: TLAiMessages): TLAiMessage[];
 
+// @public
+export function defaultApplyChange({ change, editor }: {
+    change: TLAiChange;
+    editor: Editor;
+}): void;
+
```

Comment:
Do we still need to be able to customize the apply function?

I think we do... because we'd definitely need it for cases where you want to 'preview' a change rather than applying it immediately, eg: autocomplete